src/transformers/modeling_flax_utils.py

```diff
@@ -55,6 +58,10 @@
 }
 
 
+def identity(x, **kwargs):
```

Comment:
could we maybe also remove this for now? :-)

Reply:
this is needed for the `SequenceSummary`, but maybe there is a jax/flax version that I missed (needs to have the kwargs thing)